### PR TITLE
restore encoded version of api keys

### PR DESCRIPTION
**Pull request by ggarri**
## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://github.com/consensys/quorum-key-manager/blob/master/CHANGELOG.md).

### CODE REVIEW

File: src/infra/api-key/csv/reader.go [hunk 1]
```diff
@@ -18,15 +15,14 @@ const (
 	csvSeparator         = ','
 	csvCommentsMarker    = '#'
 	csvRowLen            = 4
-	csvHashOffset        = 0
+	csvAPIKeyOffset      = 0
```
Comment (6l20) [inline src/infra/api-key/csv/reader.go:18]: I dont get why this is called ApiKeyOffset when it is actually the ApiKeyHash that is located here

File: src/infra/api-key/csv/reader.go [hunk 2]
```diff
@@ -65,14 +61,8 @@ func (r *Reader) Load(_ context.Context) (map[string]*entities.UserClaims, error
...
 
-		r.hasher.Reset()
-		_, err = r.hasher.Write([]byte(cells[csvHashOffset]))
-		if err != nil {
-			return nil, fmt.Errorf("failed to hash api key")
-		}
-
-		apiKeyHash := base64.StdEncoding.EncodeToString(r.hasher.Sum(nil))
-		claims[apiKeyHash] = &entities.UserClaims{
+		apiKeySha256 := cells[csvAPIKeyOffset]
```
Comment (6l20) [inline src/infra/api-key/csv/reader.go:64]: Related to comment above that produces this confusion here

File: src/auth/service/authenticator/authenticator.go [hunk 3]
```diff
@@ -78,10 +77,11 @@ func (authen *Authenticator) AuthenticateAPIKey(_ context.Context, apiKey []byte
 		return nil, fmt.Errorf("failed to hash api key")
 	}
 
-	claims, ok := authen.apiKeyClaims[base64.StdEncoding.EncodeToString(authen.hasher.Sum(nil))]
+	apiKeySha256 := fmt.Sprintf("%x", authen.hasher.Sum(nil))
```
Comment (darioAnongba) [inline src/auth/service/authenticator/authenticator.go:80]: why do you need `fmt.Sprintf` here? Seems that you get back []byte and you can simply do `string(authen.hasher.Sum(nil))`

**Review by darioAnongba — APPROVED** [review]

File: src/auth/service/authenticator/authenticator.go (hunk 3 above)
Comment (darioAnongba) [reply src/auth/service/authenticator/authenticator.go:80]: resolved

File: src/infra/api-key/csv/reader.go (hunk 2 above)
Comment (darioAnongba) [reply src/infra/api-key/csv/reader.go]: done

File: src/infra/api-key/csv/reader.go (hunk 1 above)
Comment (darioAnongba) [reply src/infra/api-key/csv/reader.go]: done

**Pull request merged**